scripts/client-integration-tests/tests.ts

```diff
@@ -85,3 +89,27 @@ describe("apps:sdkconfig", () => {
     expect(config.sdkConfig.appId).to.equal(appID);
   });
 });
+
+describe("database:set|get|remove", () => {
+  it("should be able to interact with the database", async () => {
+    const opts = { project: process.env.FBTOOLS_TARGET_PROJECT };
+    const path = `/${uuid()}`;
+    const data = { foo: "bar" };
+
+    await client.database.set(
+      path,
+      Object.assign({ data: JSON.stringify(data), confirm: true }, opts)
+    );
+
+    // Have to read to a file in order to get data.
+    const file = tmp.fileSync();
+
+    await client.database.get(path, Object.assign({ output: file.name }, opts));
+    expect(JSON.parse(readFileSync(file.name).toString())).to.deep.equal(data);
+
+    await client.database.remove(path, Object.assign({ confirm: true }, opts));
+
+    await client.database.get(path, Object.assign({ output: file.name }, opts));
+    expect(JSON.parse(readFileSync(file.name).toString())).to.equal(null);
```

Comment:
```suggestion
    expect(JSON.parse(readFileSync(file.name, 'utf-8'))).to.equal(null);
```

More explicit, less conversion